portal-web/test/functional/com/liferay/portalweb/macros/WorkflowMetrics.macro

```diff
@@ -111,6 +139,23 @@ definition {
 		Click(locator1 = "WorkflowMetrics#METRICS_WORKLOAD_BY_ASSIGNEE_FILTER_CLEAR_ALL");
 	}
 
+	macro create6UsersAndAssign {
+		ProductMenu.gotoPortlet(
+			category = "Content &amp; Data",
+			panel = "Site Administration",
+			portlet = "Blogs");
+
+		for (var count : list "1,2,3,4,5,6") {
+			JSONBlog.addEntry(
+				entryContent = "Blogs Entry ${count} Content",
+				entryTitle = "Blogs Entry Title ${count}");
+		}
+
+		WorkflowMetrics.executeWorkflowScript(scriptName = "add6UsersWithRoles.groovy");
```

Comment:
Can we use createUsersAndAssignInstances here? With the changes you and Michael did, does this macro work here?